src/linux/perf.cpp

```diff
@@ -125,6 +125,12 @@ class Perf : public Process<Perf>
 private:
   void execute()
   {
+    // If the locale is such that `LC_NUMERIC` uses the comma ',' as decimal
+    // separator, parsing won't work - because of unexpected number of fields
+    // and floating points format - so make sure it's set to `C`.
+    std::map<string, string> env = os::environment();
+    env["LC_ALL"] = "C";
```

Comment:
So here we set `LC_ALL` to `C` to make sure the output of `perf stat` always has `.` as the decimal
separator (e.g., `100.00` instead of `100,00`) regardless user's locale, right?

Reply:
Yep.